exporters/zipkin/model.go

```diff
@@ -197,6 +198,25 @@ func toZipkinTags(data tracesdk.ReadOnlySpan) map[string]string {
 			m[(string)(kv.Key)] = kv.Value.Emit()
 		}
 	}
+	for _, kv := range resourceAttr {
+		switch kv.Value.Type() {
+		// For slice attributes, serialize as JSON list string.
+		case attribute.BOOLSLICE:
+			json, _ := json.Marshal(kv.Value.AsBoolSlice())
+			m[(string)(kv.Key)] = (string)(json)
+		case attribute.INT64SLICE:
+			json, _ := json.Marshal(kv.Value.AsInt64Slice())
+			m[(string)(kv.Key)] = (string)(json)
+		case attribute.FLOAT64SLICE:
+			json, _ := json.Marshal(kv.Value.AsFloat64Slice())
+			m[(string)(kv.Key)] = (string)(json)
+		case attribute.STRINGSLICE:
+			json, _ := json.Marshal(kv.Value.AsStringSlice())
+			m[(string)(kv.Key)] = (string)(json)
+		default:
+			m[(string)(kv.Key)] = kv.Value.Emit()
+		}
```

Comment:
This should probably be extracted into a helper method rather than repeated here.